<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Mainwindow</class>
 <widget class="QDialog" name="Mainwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1142</width>
    <height>593</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>741</width>
     <height>441</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLineEdit" name="pesquisarlineEdit"/>
        </item>
        <item>
         <widget class="QPushButton" name="pesquisaBtn">
          <property name="text">
           <string>Pesquisar</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableWidget" name="tableWidget"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>30</y>
     <width>79</width>
     <height>331</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QPushButton" name="listarBtn">
      <property name="text">
       <string>Listar Todos</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="addBtn">
      <property name="text">
       <string>Adicionar</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="editarBtn">
      <property name="text">
       <string>Editar</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_5">
      <property name="text">
       <string>Excluir</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="configBtn">
      <property name="text">
       <string>Configurações</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="sairBtn">
      <property name="text">
       <string>Sair</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
